apps/nuxt3-ssr/components/content/ContentBlockIntro.vue

```diff
@@ -83,35 +95,43 @@ const notificationMessage = ref("");
 const timeoutInMills = ref(3000);
 
 const submitForm = async () => {
-  const senderName = fields.find((field) => field.name === "senderName");
-  const senderEmail = fields.find((field) => field.name === "senderEmail");
-  const senderMessage = fields.find((field) => field.name === "senderMessage");
   // Validate form fields
-
-  if (!senderEmail.fieldValue) {
-    senderEmail.hasError = true;
-    senderEmail.message = "Please enter a valid email address";
+  if (!fields.senderEmail.fieldValue) {
```

Comment:
In the constants file of the components lib is a regex for validating email addresses, might be an idea to apply it here?